CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required( VERSION 3.16 )
+project( T8CODE DESCRIPTION "Parallel algorithms and data structures for tree-based AMR with arbitrary element shapes." LANGUAGES C CXX )
+include( CTest )
+
+option( T8CODE_ENABLE_MPI "" ON )
+option( T8CODE_BUILD_TESTS "" ON )
+option( T8CODE_BUILD_TUTORIALS "" ON )
```

Comment:
How about also adding the `example` directory and build all examples?

Reply:
Yes and I realized this morning that I have not ported all tutorials too. It's on my plate for today 😉

Reply:
The examples are now ported